contrib/kubespray/quick-start/services-configuration.yaml.template

```diff
@@ -0,0 +1,197 @@
+  #  # AZURE: https://login.microsoftonline.com/{tenantID}/v2.0/.well-known/openid-configuration
+  #  # China: https://login.partner.microsoftonline.cn/{tenantID}/v2.0/.well-known/openid-configuration
+  #  # Germany: https://login.microsoftonline.de/{tenantID}/v2.0/.well-known/openid-configuration
+  #  wellKnownURL: https://login.microsoftonline.com/{tenantID}/v2.0/.well-known/openid-configuration
+  #
+  #  # Required
+  #  tenantID: ${tenat_id}
+  #
+  #  # Required, the client ID of your app in AAD
+  #  clientID: ${your_client_id}
+  #
+  #  # Required if `responseType` is 'code', 'id_token code' or 'code id_token'.
+  #  # If app key contains '\', replace it with '\\'.
+  #  clientSecret: '${your_client_secret}'
+  #
+  #  # Optional. The lifetime of nonce in session or cookie, the default value is 3600 (seconds).
+  #  nonceLifetime: null
+  #
+  #  # Optional. The max amount of nonce saved in session or cookie, the default value is 10.
+  #  nonceMaxAmount: 5
+  #
+  #  # Optional. The clock skew allowed in token validation, the default value is 300 seconds.
+  #  clockSkew: null
+  #
+  #  # Optional.
+  #  # If you want to get access_token for a specific resource, you can provide the resource here; otherwise,
+  #  # set the value to null.
+  #  # Note that in order to get access_token, the responseType must be 'code', 'code id_token' or 'id_token code'.
+  #  resourceURL: 'https://graph.windows.net'
+
+  group-manager:
+    # basic: If you set group-data-source as the value basic, admin should manually modify user's grouplist.
+    # winbind: If you set group-data-source as the value winbind, the user's grouplist will get from winbind server based on your configuration.
+    group-data-source: basic
+
+    # If you set winbind as your data source, you should configure this configuration.
+    # winbind-server-address: xxxxxxx
+
+    # Admin group name and its user list
+    admin-group:
+      groupname: admingroup
+      description: "admin's group"
+      externalName: ""
+      extension:
+        acls:
+          virtualClusters: []
+          admin: true
+
+    # Group for default vc.
+    # For yarn default queue hack.
+    default-group:
+      groupname: default
+      description: "group for default vc"
+      externalName: ""
+      extension:
+        acls:
+          virtualClusters: ["default"]
+          admin: false
+
+  #  # If the following groups are not in the data store, it will be created by default.
+  #  grouplist:
+  #    - groupname: forexample
+  #      # internal name
+  #      description: forexample
+  #      # description of the group
+  #      externalName: ""
+  #      # external name, it should be set if your group-data-source is winbind. And the name will be used to query and match the group from
+  #      # the result of winbind. If the group-data-source is basic, this field is useless.
+
+
+hivedscheduler:
+  config: |
```

Comment:
Add comment to link hived:
[https://github.com/microsoft/pai/blob/master/subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md#Config](https://github.com/microsoft/pai/blob/master/subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/doc/user-manual.md#Config)

Reply:
Done

Reply:
It is not commented:

#hivedscheduler:

Reply:
Cool，Done，